src/renderers/dom/shared/dangerousStyleValue.js

```diff
@@ -23,9 +25,10 @@ var isUnitlessNumber = CSSProperty.isUnitlessNumber;
  *
  * @param {string} name CSS property name such as `topMargin`.
  * @param {*} value CSS property value such as `10px`.
+ * @param {ReactDOMComponent} component
  * @return {string} Normalized style value with dimensions applied.
  */
-function dangerousStyleValue(name, value) {
+function dangerousStyleValue(name, value, component) {
```

Comment:
We should probably take in a component name, since some components will be stateless and won't have instances but should still have names (I think).